molecule/default/molecule.yml

```diff
@@ -8,6 +9,9 @@ lint:
   options:
     config-file: molecule/default/yamllint.yml
 platforms:
+  - name: centos-6
+    image: linuxsystemroles/centos-6
+    privileged: true
```

Comment:
This role does not support RHEL-6 at all, but I don't know if it should be listed as a platform or not.